db/team_test.go

```diff
@@ -60,6 +60,16 @@ var _ = Describe("Team", func() {
 		})
 	})
 
+	Describe("Rename", func() {
+		JustBeforeEach(func() {
+			Expect(team.Rename("oopsies")).To(Succeed())
+		})
+
+		It("renames the team", func() {
+			Expect(team.Name()).To(Equal("oopsies"))
```

Comment:
This would pass even if it didn't persist the change; maybe it should test against a newly fetched team instead? Then you wouldn't have to update the `t.name` field; I'd like those fields to always come from the database and not be changed by anything but a `.Reload` method (which currently exists on other objects, but not Team).